app/controllers/supplejack_api/records_controller.rb

```diff
@@ -127,6 +127,10 @@ def search_params
       params[:search]&.to_unsafe_h
     end
 
+    def next_previous_search_params
+      params.require(:search).permit([:page, :per_page, :record_type, :text, { and: {} }, { or: {} }]).to_unsafe_h
```

Comment:
Do you still need `.to_unsafe_h`?

Reply:
I've kept the same shape as the previous params, I'm not too sure how this param is being used downstream :(